pi-apps-terminal-bash-edition.sh

```diff
@@ -148,106 +90,154 @@ while [ "$1" != "" ]; do
+			list_apps corrupted
 			exit $?
 		;;
 		list-all)
 			#list all the apps
-			list-all
+			list_apps cpu_installable
 			exit $?
 		;;
 		search)
 			shift
-			args="$*"
-			#search apps
-			search "$args"
+			echo
+			while read -r line; do
+				[[ -z "$line" ]] && continue
+				echo -e "${bold}${inverted}${light_blue}${underline}$line${normal}\n"
+				echo -e "${green}$(cat "$DIRECTORY/apps/$line/description" || echo "No description available")${normal}\n"
+			done < <(app_search "$1" 2>/dev/null)
 			exit $?
 		;;
-		update-apps)
-			#update all pi-apps apps
-			$PI_APPS_DIR/manage update-all
-			exit $?
-			;;
 		update)
-			#update all pi-apps
-			$PI_APPS_DIR/updater cli
+			#update pi-apps
+			shift
+			if [ "$*" == "cli" ] || [ "$*" == "gui" ] || [ "$*" == "cli-yes" ] || [ "$*" == "gui-yes" ] || [ "$*" == "" ]; then
+				"$DIRECTORY/updater" "$*" 
+			else
+				error "Argument '$*' not available. \n    Accepted arguments: [gui|cli|gui-yes|cli-yes]"
+			fi
 			exit $?
 		;;
-		website)
+		info)
 			shift
-			[[ "$@" == "" ]] && error "'website' option passed, but no app provided!"
-			args="$*"
-			#print the website of a app
-			get-website "$args" 2>/dev/null
-			if [[ "$website_error" == "1" ]]; then
-				echo -e "${red}${bold}ERROR:${normal}${red} There is no app called ${light_red}'$@'${red}!${normal}"
-				exit 1
+			
+			if [ -z "$*" ]; then
+				error "No app provided."
 			else
-				echo -e "${cyan}${inverted}$@'s website:${normal}"
-				echo -e "${bold}$website${normal}"
-				exit 0
+				app_name="$*"
+			fi
+				
+			if ! [ -d "$DIRECTORY/apps/$app_name" ]; then
+				error "App not found."
+			fi
+			
+			description="$(cat "${DIRECTORY}/apps/$app_name/description" || echo 'Description unavailable')"
+
+			abovetext="
+- Current status: $(app_status "$app_name") | sed 's/corrupted/corrupted (installation failed)/g' | sed 's/disabled/disabled (installation is prevented on your system)/g')"
+			
+			if [ -f "${DIRECTORY}/apps/$app_name/website" ];then
+				#show website if it exists
+				abovetext="$abovetext
+- Website: $(head -n1 < "${DIRECTORY}/apps/$app_name/website")"
+
+			fi
+
+			if [ -z "$clicklist" ];then
+				clicklist="$(usercount)"
+			fi
+
+			usercount="$(echo "$clicklist" | grep " $app_name"'$' | awk '{print $1}' | head -n1)"
+			if [ -n "$usercount" ] && [ "$usercount" -gt 20 ];then
+				abovetext="$abovetext
+- $(printf "%'d" "$usercount") users"
+
+				if [ "$usercount" -ge 1500 ] && [ "$usercount" -lt 10000 ];then
+				  #if a lot of users, add an exclamation point!
+				  abovetext="${abovetext}!"
+				elif [ "$usercount" -ge 10000 ];then
+				  #if a crazy number of users, add two exclamation points!
+				  abovetext="${abovetext}!!"
+				fi
 			fi
+			
+			status_green "\n\e[4m$app_name"
+			status "$abovetext\n"
+			echo -e "$description\n" 
+			exit 0
+		;;
+		import)
+			shift
+			"$DIRECTORY/etc/import-app" "$*"
+			exit $?
```

Comment:
I don't want any GUI only functions.

Reply:
`import-app` is available in CLI. I've made it to exit if no argument is provided in https://github.com/cycool29/PiApps-terminal_bash-edition/commit/04926f490147be8d829cdacfa23303ede15735c6.

Reply:
For now, I will remove the `import` first and I will try to make a full CLI version of `import-app` (and maybe `settings` and `createapp`), like `updater`.

Once that is done, I will open another PR here.